README.md

```diff
@@ -416,6 +417,24 @@ const powerset = arr =>
 // powerset([1,2]) -> [[], [1], [2], [2,1]]
 ```
 
```

Comment:
You don't have to make any changes to the README or rebuild, I'll handle that.

Reply:
Sorry, how do I revert these changes?

Reply:
It's alright, I'll take it from here, don't worry about them.